From 4075ed9247ad1061efe5438373b7d82230639c77 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Mon, 4 Feb 2013 13:12:28 +0000
Subject: [PATCH] build: Add src/guestfs-internal-{all,frontend}.h to
 EXTRA_DIST.

This fixes commit ec3b75e5ffd1abea1400f6609ae9c61eab0dfa71.
---
 src/Makefile.am | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/Makefile.am b/src/Makefile.am
index acdbe74..e344190 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -128,6 +128,8 @@ libguestfs_la_LDFLAGS += $(VERSION_SCRIPT_FLAGS)$(srcdir)/libguestfs.syms
 libguestfs_la_SOURCES = \
 	guestfs.h \
 	guestfs-internal.h \
+	guestfs-internal-all.h \
+	guestfs-internal-frontend.h \
 	guestfs_protocol.h \
 	actions-0.c \
 	actions-1.c \
-- 
1.8.1

